articles/security/azure-operational-security-best-practices.md

```diff
@@ -53,38 +53,38 @@ Customers build an end-to-end network in Azure by combining network resources li
 
-**Best practice**: Automate the build and deployment of services.   
+**Best practice**: Automate the build and deployment of services.  
 **Detail**: [Infrastructure as code](https://en.wikipedia.org/wiki/Infrastructure_as_Code) is a set of techniques and practices that help IT pros remove the burden of day-to-day build and management of modular infrastructure. It enables IT pros to build and maintain their modern server environment in a way that’s like how software developers build and maintain application code.
 
 You can use [Azure Resource Manager](https://azure.microsoft.com/documentation/articles/resource-group-authoring-templates/) to provision your applications by using a declarative template. In a single template, you can deploy multiple services along with their dependencies. You use the same template to repeatedly deploy your application in every stage of the application lifecycle.
 
-**Best practice**: Automatically build and deploy to Azure web apps or cloud services.   
-**Detail**: You can configure your Visual Studio Team Services (VSTS) team projects to [automatically build and deploy](https://www.visualstudio.com/docs/build/overview) to Azure web apps or cloud services. VSTS automatically deploys the binaries after doing a build to Azure after every code check-in. The package build process is equivalent to the Package command in Visual Studio, and the publishing steps are equivalent to the Publish command in Visual Studio.
+**Best practice**: Automatically build and deploy to Azure web apps or cloud services.  
+**Detail**: You can configure your Azure DevOps Projects to [automatically build and deploy](https://www.visualstudio.com/docs/build/overview) to Azure web apps or cloud services. VSTS automatically deploys the binaries after doing a build to Azure after every code check-in. The package build process is equivalent to the Package command in Visual Studio, and the publishing steps are equivalent to the Publish command in Visual Studio.
 
-**Best practice**: Automate release management.   
+**Best practice**: Automate release management.  
 **Detail**: Visual Studio [Release Management](https://msdn.microsoft.com/library/vs/alm/release/overview) is a solution for automating multiple-stage deployment and managing the release process. Create managed continuous deployment pipelines to release quickly, easily, and often. With Release Management, you can automate your release process, and you can have predefined approval workflows. Deploy on-premises and to the cloud, extend, and customize as required.
 
-**Best practice**: Check your app's performance before you launch it or deploy updates to production.   
+**Best practice**: Check your app's performance before you launch it or deploy updates to production.  
 **Detail**: Run cloud-based [load tests](https://www.visualstudio.com/docs/test/performance-testing/getting-started/getting-started-with-performance-testing) by using VSTS to:
 
```

Comment:
I think this should go to https://docs.microsoft.com/en-us/azure/devops/test/load-test/app-service-web-app-performance-test?view=vsts and say "by using Azure Test Plans" @alexhomer1?